### ISSUE TITLE
Remove duplicate CSS

### ISSUE THREAD

**Issue by KenTuei** [ivycommunity001/Momentum-Collective#4]
**Task**
- Fix incorrect hyperlinks across site
- Remove duplicate CSS
- Add missing alt text

 **Deliverables**
Buttons and links fixed across site, basic
accessibility updated
